tests/e2e/chain/config.go

```diff
@@ -416,6 +317,90 @@ func initGenesis(c *internalChain, votingPeriod time.Duration) error {
 	return nil
 }
 
+func updateBankGenesis(bankGenState *banktypes.GenesisState) {
+	bankGenState.DenomMetadata = append(bankGenState.DenomMetadata, banktypes.Metadata{
+		Description: "An example stable token",
+		Display:     OsmoDenom,
+		Base:        OsmoDenom,
+		Symbol:      OsmoDenom,
+		Name:        OsmoDenom,
+		DenomUnits: []*banktypes.DenomUnit{
+			{
+				Denom:    OsmoDenom,
+				Exponent: 0,
+			},
+		},
+	})
+}
+
+func updateStakeGenesis(stakeGenState *staketypes.GenesisState) {
+	stakeGenState.Params = staketypes.Params{
+		BondDenom:         OsmoDenom,
+		MaxValidators:     100,
+		MaxEntries:        7,
+		HistoricalEntries: 10000,
+		UnbondingTime:     240000000000,
+		MinCommissionRate: sdk.ZeroDec(),
+	}
+}
+
+func updatePoolIncentiveGenesis(pooliGenState *poolitypes.GenesisState) {
+	pooliGenState.LockableDurations = []time.Duration{
+		time.Second * 120,
+		time.Second * 180,
+		time.Second * 240,
+	}
+	pooliGenState.Params = poolitypes.Params{
+		MintedDenom: OsmoDenom,
+	}
+}
+
+func updateIncentivesGenesis(incentivesGenState *incentivestypes.GenesisState) {
+	incentivesGenState.LockableDurations = []time.Duration{
+		time.Second,
+		time.Second * 120,
+		time.Second * 180,
+		time.Second * 240,
+	}
+	incentivesGenState.Params = incentivestypes.Params{
+		DistrEpochIdentifier: "day",
+	}
+}
+
+func updateMintGenesis(mintGenState *minttypes.GenesisState) {
+	mintGenState.Params.MintDenom = OsmoDenom
+	mintGenState.Params.EpochIdentifier = "day"
+}
+
+func updateGenUtilGenesis(c *internalChain) func(*genutiltypes.GenesisState) {
```

Comment:
There is a PR in progress that aims to allow initializing state for single full nodes to be added to existing chains. Just linking it to help with reconciling the differences after one of these is merged: https://github.com/osmosis-labs/osmosis/pull/1587